htdocs/js/ui/init.js

```diff
@@ -1,4 +1,6 @@
 RCloud.UI.init = function() {
+
+
     $("#fork-notebook").click(function() {
```

Comment:
just a nit, but please look out for added newlines. it just adds noise to the changelog.